lib/net.js

```diff
@@ -1107,6 +1107,19 @@ function Server(options, connectionListener) {
     configurable: true, enumerable: false
   });
 
+  Object.defineProperty(this, 'listening', {
+    get: function() {
+      return !! self._handle;
+    },
+    set: function() {
```

Comment:
I'd simply omit the `set` here. There's no reason to add a new throw as this can easily be marked as read only in the documentation